cmd/mothership/mothership/start/scheduler.go

```diff
@@ -42,6 +43,10 @@ func startScheduler(ctx context.Context, o *Options, configFile string) error {
 			OperationsWatchInterval: 30 * time.Second,
 			OrphanOperationTimeout:  o.OrphanOperationTimeout,
 		}).
+		WithCleanerConfig(&service.CleanerConfig{
+			PurgeEntitiesOlderThan: 14 * 24 * time.Hour, // 14 days
```

Comment:
Would be good to make the purge-interval configurable os CLI flag. Using 14days as default is good.

Reply:
Done!